connector/google/google.go

```diff
@@ -49,6 +49,9 @@ type Config struct {
 	// The email of a GSuite super user which the service account will impersonate
 	// when listing groups
 	AdminEmail string
+
+	// If this field is true, both direct and indirect group memberships will be fetched
+	GroupsIndirectFetch bool `json:"groupsIndirectFetch"`
```

Comment:
I think it would be better to have a more explicit naming here.

Eg. `FetchIndirectGroupMemberships` or `FetchTransitiveGroupMembership`, WDYT?